conans/model/info.py

```diff
@@ -328,6 +328,7 @@ class ConanInfo:
     def __init__(self, settings=None, options=None, reqs_info=None, build_requires_info=None,
                  python_requires=None, conf=None, config_version=None):
         self.invalid = None
+        self.cant_build = False  # It will set to a str with a reason if the validate_build() fails
```

Comment:
Dont we want to serialize/clone this when calling the appropiate methods?

Reply:
This was already being serialized as some "root" information of the node, not as ``cpp_info`` field, I thought it is better to not break the serialization output, so it keeps being serialized above in ``result["invalid_build"] = getattr(getattr(self.conanfile, "info", None), "cant_build", False)``